build.sbt

```diff
@@ -4,8 +4,11 @@ version := "1.0-SNAPSHOT"
 
 scalaVersion := "2.11.8"
 
+enablePlugins(GatlingPlugin)
+
 libraryDependencies += "io.gatling" % "gatling-core" % "2.2.0"
 libraryDependencies += "io.gatling" % "gatling-http" % "2.2.0"
-
+libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % "2.2.0"
```

Comment:
I am reading the AFFERO license....

gatling.highcharts fall on the definition of a system library : 

> The "System Libraries" of an executable work include anything, other than the
work as a whole, that (a) is included in the normal form of packaging a Major
Component, but which is not part of that Major Component, and (b) serves only
to enable use of the work with that Major Component, or to implement a Standard
Interface for which an implementation is available to the public in source code
form. A "Major Component", in this context, means a major essential component
(kernel, window system, and so on) of the specific operating system (if any) on
which the executable work runs, or a compiler used to produce the work, or an
object code interpreter used to run it.

It is stated that : 

> The "Corresponding Source" [...] does not include the work's System Libraries